back/src/Model/Group.ts

```diff
@@ -106,6 +106,16 @@ export class Group implements Movable {
         return this.users.size <= 1;
     }
 
+    includes(user: User): boolean {
+        let found = false;
+        this.users.forEach((currentUser: User) => {
+            if (currentUser.name === user.name) {
```

Comment:
```suggestion
            if (currentUser.id === user.id) {
```

We can switch to using user ID here too.